packages/wallet/src/KeyManagement/errors/GenericError.ts

```diff
@@ -0,0 +1,19 @@
+import { CustomError } from 'ts-custom-error';
+
+export enum GenericErrorType {
+  NO_METHOD = 'NO_METHOD'
+}
+
+export class GenericError extends CustomError {
+  getMessage(errorType: GenericErrorType): string {
+    if (errorType === GenericErrorType.NO_METHOD) {
+      return 'Method not implemented';
```

Comment:
There is already a "NotImplementedError" in `core` package, please use that instead